prime-angular-frontend/src/app/modules/site-registration/pages/organization-signing-authority-page/organization-signing-authority-page.component.ts

```diff
@@ -171,12 +179,20 @@ export class OrganizationSigningAuthorityPageComponent extends AbstractEnrolment
 
     if (redirectPath) {
       routePath = [redirectPath, SiteRoutes.SITE_REVIEW];
-    } else {
+    }
+    else if (this.route.snapshot.queryParams.claimOrg === 'true') {
+      if (organization) {
+        routePath = ['../../', SiteRoutes.ORGANIZATION_CLAIM_CONFIRMATION];
+      }
+      else {
+        this.toastService.openErrorToast('Could not claim the organization.');
```

Comment:
The message indicating that it is already claimed should appear in the page, and not in a toast that disappears.  We should move the claim page after the signing authority is created so they already exist, and we don't need to use queryParams to indicate state or potentially lose data if they refresh.